pyannote/audio/augmentation/registry.py

```diff
@@ -94,41 +89,39 @@ def forward(self, spectrograms):
 
     """
 
-    wrapped_augmentation = wrap_augmentation(augmentation, module, when=when)
```

Comment:
What are the motivation for getting rid of this wrapper? 

I thought it was needed to pass `sample_rate` to `torch-audiomentations` transform `.forward`.

Reply:
This isn't needed anymore, you can specify the sample rate in the init

Reply:
I understand but that also means that you need to repeat `sample_rate` multiple times (once for the model, once for the augmentation). With the wrapper, you only have to do it once for the model -- ensuring sample rate consistency automatically.

Reply:
That is true, the main reason I pulled it out was because of cyclic dependencies when importing the registeration method in the core.task/model files. I'll see if I can get around that, otherwise we could iterate of the modules in task.augmentation and set the sample rate there.

Reply:
I wasn't able to resolve the cyclic dependencies but I created a method that checks the augmentations in the setup and gives them a sample rate if they don't have one.